normandy/health/tests/test_api.py

```diff
@@ -26,3 +28,53 @@ def test_lbheartbeat_makes_no_db_queries(client):
         res = client.get('/__lbheartbeat__')
         assert res.status_code == 200
     assert len(queries) == 0
+
+
+class TestGetVersionInfo(object):
+
+    @pytest.fixture
+    def version_info_dir(self, tmpdir, settings):
+        views._tag = None
+        views._commit = None
+        settings.BASE_DIR = tmpdir.strpath
+        return tmpdir.mkdir('__version__')
+
+    @pytest.fixture
+    def commit_file(self, version_info_dir):
+        f = version_info_dir.join('commit')
+        f.write('bc685e4be05a182ae819990509c92affa3d882ab\n')
```

Comment:
I've never been able to come up with a good way to pass parameters to pytest fixtures.